src/physics.js

```diff
@@ -0,0 +1,58 @@
+import Vec2 from 'vec2';
+
+export default class Physics {
+    constructor(cameraBody, distanceFactor = 1.5, timeStep = 10, gravity = 9.82) {
+        this.cameraBody = cameraBody;
+        this.physicsBodyArr = new Array();
+        this.distanceFactor = distanceFactor;
+        this.timeStep = timeStep;
+        this.gravity = gravity; //units/(s^2)
+    }
+
+    addBody(physicsBody) {
+        this.physicsBodyArr.push(physicsBody);
+        this.cameraSort();
+    }
+
+    resolveView() {
+        this.cameraSort();
+
+        for (const physicsBody of this.physicsBodyArr) {
+            const physicsWindow = this.cameraBody.polygon.scale(this.distanceFactor, Vec2(this.cameraBody.position), true);
+            if (physicsWindow.containsPoint(Vec2(physicsBody.position))) {
+                this.resolveBody(physicsBody, physicsWindow);
+            }
+
+            physicsBody.inView = this.inView(physicsBody);
+        }
+
+        return this;
+    }
+
+    resolveBody(physicsBody) {
+        const tempBody = physicsBody.clone;
```

Comment:
Is there a way we can do this without making a clone? That seems inefficient... 

For now, I'd just put a TODO comment here maybe noting possible inefficiencies, and we can investigate later if this becomes a performance problem. If it become a problem, we'll put it in the backlog.